tsl/test/isolation/specs/compressed_copy.spec

```diff
@@ -5,6 +5,7 @@
 setup {
   CREATE TABLE metrics (time timestamptz not null, device text, value float) WITH (tsdb.hypertable, tsdb.partition_column = 'time');
 	INSERT INTO metrics (time, device, value) VALUES ('2025-01-01 00:00:00', 'd1', 0.1);
+  SELECT compress_chunk(show_chunks('metrics'));
```

Comment:
I think precompressing here fundamentally changes the test because you will not enter the path in `chunk_tuple_routing` where you dynamically create a new compressed chunk: https://github.com/timescale/timescaledb/blob/3445b9bb51f0b290f7ab11a4896f24c2cbc11790/src/chunk_tuple_routing.c#L135

You should, in fact, probably test both: 

1. Compressed chunk exists when copy starts
2. Compressed chunk does **_not_** exist when copy starts

Reply:
if the chunk is created in s1 then s2 will have to wait till s1 commits, that is true for all parallel chunk operations